pkg/providers/instancetype/instancetype.go

```diff
@@ -276,7 +276,7 @@ func computeCapacity(ctx context.Context, sku *skewer.SKU, kc *corev1beta1.Kubel
 // gpuNvidiaCount returns the number of Nvidia GPUs in the SKU. Currently nvidia is the only gpu manufacturer we support.
 func gpuNvidiaCount(sku *skewer.SKU) *resource.Quantity {
 	count, err := sku.GPU()
-	if err != nil || !utils.IsNvidiaEnabledSKU(sku.GetName()) {
+	if err != nil || !agentbakercommon.IsNvidiaEnabledSKU(sku.GetName()) {
```

Comment:
I saw ganesh update a non shared library again. Can we try going forward to update in the shared package and bump the version when making those changes?

Reply:
@lilypan26  could you please take a look at this?

Reply:
Yes, I will let Ganesh know and sync the changes.